### PR TITLE
added documentation for connecting firefly to a remote fabric chaincode

### PR DESCRIPTION
**Pull request by satoshi-u**
Signed-off-by: Sarthak Joshi <sarthakjoshi.in@gmail.com>

### CODE REVIEW

File: docs/tutorials/fabric_remote.md [hunk 1]
```diff
@@ -0,0 +1,425 @@
+---
+layout: default
+title: Work with remote fabric network
+parent: pages.tutorials
+nav_order:
+---
+
+# Work with remote Hyperledger Fabric Network
+
```
Comment (nguyer) [inline docs/tutorials/fabric_remote.md:9]:
```suggestion
```
Removing this extra line of whitespace will achieve the desired effect of not having the page title in the page's table of contents below

File: docs/tutorials/fabric_remote.md [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+Web UI for member '0': http://127.0.0.1:5000/ui
+Sandbox UI for member '0': http://127.0.0.1:5108
+
+To see logs for your stack run:
+
+ff logs my-remote-fabric
+
+```
+
+## Define and Broadcast FireFly Interface Document for your chaincode
```
Comment (nguyer) [inline docs/tutorials/fabric_remote.md:250]: It looks like most of this section and the rest of this guide are copied from this page: https://hyperledger.github.io/firefly/tutorials/custom_contracts/fabric.html

I think we can have a short description of what you need to do next here, but then link to that page for the details. That way we only have one page we need to update / add to when we need to talk about using custom contracts with Fabric networks.

If you feel there is information missing from that tutorial (I see you added some detail about the Sandbox here which is great!) please feel free to update the other tutorial as well.

File: docs/tutorials/fabric_remote.md [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+title: Work with remote fabric network
+parent: pages.tutorials
+nav_order:
+---
+
+# Work with remote Hyperledger Fabric Network
+
+{: .no_toc }
+
+This guide describes the steps to use FireFly to interact with a chaincode deployed to an external Hyperledger Fabric blockchain network (not created by Firefly) in order to submit transactions and query for states.
```
Comment (nguyer) [inline docs/tutorials/fabric_remote.md:12]: I think adding a note at the top of this tutorial to set a bit more context would be helpful to people. While this is a fantastic guide for helping people connect a local development environment to a pre-existing Fabric network, we don't want people to get the idea that the FireFly CLI should be used for setting up production networks. I might suggest something like this:

```suggestion
The FireFly CLI makes it quick and easy to create an entire FireFly development environment on your local machine, including a new blockchain from scratch. However, sometimes a developer may want to connect their local FireFly development environment to a Fabric network that already exists on their machine or elsewhere. This guide describes the steps to connect FireFly to an external Hyperledger Fabric blockchain network (not created by the FireFly CLI), including interacting with a custom chaincode in order to submit transactions and query for state.
```

**Review by nguyer — CHANGES_REQUESTED** [review]
Thank you very much for contributing this, @satoshi-u! This is really useful and I definitely think it will help other people who want to do the same thing. I have a few suggestions for minor changes and then I would love to get this merged.

File: docs/tutorials/fabric_remote.md (hunk 2 above)
Comment (satoshi-u) [reply docs/tutorials/fabric_remote.md]: Addressed.

**Comment by satoshi-u** [comment]
> Thank you very much for contributing this, @satoshi-u! This is really useful and I definitely think it will help other people who want to do the same thing. I have a few suggestions for minor changes and then I would love to get this merged.

Happy to contribute! I have incorporated your suggestions in my PR. Thanks!

File: docs/tutorials/fabric_remote.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+
+ff logs my-remote-fabric
+
+```
+
+## Integrate your FireFly stack with your Fabric chaincode 
+
+Once your stack is up and running, you can define and broadcast FireFly Interface Document for your remote fabric chaincode. You can refer to [Broadcast the Contract Interface(Fabric)](http://localhost:4000/firefly/tutorials/custom_contracts/fabric.html#broadcast-the-contract-interface) guide for this.
+
+You can also now create an HTTP API for your fabric chaincode which will help you easily query/invoke all your chaincode methods with your organizational context that you set up in fabconnect. You can refer to [Create an HTTP API for the contract(Fabric)](http://localhost:4000/firefly/tutorials/custom_contracts/fabric.html#broadcast-the-contract-interface) guide for this.
```
Comment (nguyer) [inline docs/tutorials/fabric_remote.md:253]:
```suggestion
You can also now create an HTTP API for your fabric chaincode which will help you easily query/invoke all your chaincode methods with your organizational context that you set up in fabconnect. You can refer to [Create an HTTP API for the contract(Fabric)](./custom_contracts/fabric.html#broadcast-the-contract-interface) guide for this.
```

These need to be relative links so they will link correctly when the docs site is published to GitHub pages.

File: docs/tutorials/fabric_remote.md [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+
+To view the OpenAPI spec for your contract, or to submit transactions, query for states and listen for events, you can further refer to [Work with Hyperledger Fabric chaincodes](http://localhost:4000/firefly/tutorials/custom_contracts/fabric.html#work-with-hyperledger-fabric-chaincodes) guide.
```
Comment (nguyer) [inline docs/tutorials/fabric_remote.md:255]:
```suggestion
To view the OpenAPI spec for your contract, or to submit transactions, query for states and listen for events, you can further refer to [Work with Hyperledger Fabric chaincodes](./custom_contracts/fabric.html#work-with-hyperledger-fabric-chaincodes) guide.
```

**Review by nguyer — CHANGES_REQUESTED** [review]
This looks good! Just one thing to fix with the relative links, then I'm happy to merge.

**Comment by satoshi-u** [comment]
> This looks good! Just one thing to fix with the relative links, then I'm happy to merge.

done.

**Review by nguyer — APPROVED** [review]

**Pull request merged**